pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -489,15 +489,23 @@ func (k *Kubernetes) ConfigEnvs(name string, service kobject.ServiceConfig) []ap
 // CreateKubernetesObjects generates a Kubernetes artifact for each input type service
 func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) []runtime.Object {
 	var objects []runtime.Object
+	var replica int
+	if opt.Replicas > 1 {
```

Comment:
you mean something like this,

`kompose convert  --replicas=1  --replica-set`

I am confused because, `--replicas` is int flag and what you are suggesting is boolean flag